docs/dashboards/chart-builder.md

```diff
@@ -226,3 +226,40 @@ of a given series is stacked. This is very useful for area and bar charts. Below
 a comparison of stacked and non-stacked bar charts.
 
 ![Chart builder styles](../images/dashboards/chart-builder-stacked.png)
+
+## Heatbar
+
+Heatbar allows you to show a heatmap above a time-series chart. You can style any series as a heatbar, but it is especially suitable for events and logs. For example, a heatbar of alerts above the rate of network transactions.
```

Comment:
We use the words heatmap and heatbar and don't explain what the relationship is.  Could we do that so that one is not confused?